### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by kito99**
Added info about usage with an existing org account.

### CODE REVIEW

**Review by mraible — APPROVED** [review]

File: okta-hosted-login/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ You also need to gather the following information from the Okta Developer Consol
 
 - **Client ID** and **Client Secret** - These can be found on the "General" tab of the Web application that you created earlier in the Okta Developer Console.
 
-- **Issuer** - This is the URL of the authorization server that will perform authentication.  All Developer Accounts have a "default" authorization server.  The issuer is a combination of your Org URL (found in the upper right of the console home page) and `/oauth2/default`. For example, `https://dev-1234.oktapreview.com/oauth2/default`.
+- **Issuer** - This is the URL of the authorization server that will perform authentication.  All Developer Accounts have a "default" authorization server.  The issuer is a combination of your Org URL (found in the upper right of the console home page) and `/oauth2/default`. For example, `https://dev-1234.oktapreview.com/oauth2/default`. If you have an Org Account, this will just be the Org URL (i.e. `https://my-org.oktapreview.com`).
```
Comment (bdemers) [inline okta-hosted-login/README.md:30]: Thanks @kito99!

This isn't 100% correct. The `/oauth2/default` is actually an Okta "Custom Authorization Server".  As mentioned in this line, the default one created for developer accounts is named `default`.

As you mentioned just using the Okta Org URL works too, but it's a different feature set: 
The TL;DR is using the `/oauth2/<id>` allows you to create custom scopes, and treat your OAuth Access tokens as JWTs. 

https://developer.okta.com/docs/concepts/auth-servers/#custom-authorization-server-discovery-endpoints

I think we can add more description around this though to help with any confusion.  

cc @arvindkrishnakumar-okta

**Review by bdemers — CHANGES_REQUESTED** [review]
@arvindkrishnakumar-okta we probably need to take a pass through most of the Okta libs and help clarify this. As mentioned by @kito99, you don't actually need an Okta Developer Account, but that changes what options are available for the issuer uri

**Comment by arvindkrishnakumar-okta** [comment]
@kito99 [Here](https://github.com/okta/samples-java-spring/pull/102) is a PR that adds more clarity to the README on this subject.

**Comment by arvindkrishnakumar-okta** [comment]
Closing this in favor of #102

**Pull request closed without merging**